pkg/porter/list.go

```diff
@@ -16,6 +17,40 @@ import (
 type ListOptions struct {
 	printer.PrintOptions
 	Namespace string
+	Name      string
+	Labels    []string
+}
+
+func (o *ListOptions) Validate(args []string) error {
+	if len(args) == 1 {
+		o.Name = args[0]
+	} else if len(args) > 1 {
+		return errors.Errorf("only one positional argument may be specified, the installation name, but multiple were received: %s", args)
```

Comment:
Do we need this same check for the params/creds list commands?

e.g.
```
 $ porter params list foo foo
NAMESPACE   NAME   MODIFIED
```

Reply:
I'll switch to a --name flag so that we don't need this check at all.